assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -233,4 +281,11 @@ function Carousel() {
   );
 }
 
+const Li = styled.li`
```

Comment:
Why is this defined down here? I would define this at the top with the rest of the styled components.

Reply:
Yes, things should be kept together. However, this is a precedent for declaring all styled components in the bottom of the file, as they're less important than the actual functional component in a file and thus less cluttering. If we do so, we should of course do so on a project-wide basis. Another option is to create a `foo.styles.js` next to every `foo.js` with all the styled components.

If we decided to do either, we could probably find some eslint tools to help us remember it.

Reply:
Then let's discuss this in a dedicated ticket to specify this for the whole project instead of just adding this change in 1 place. Happy to put this in writing after that and possibly enforce using ESLint if possible.

Reply:
Right, I will move it.

Reply:
> declaring all styled components in the bottom of the file, as they're less important than the actual functional component in a file and thus less cluttering

I either did this^ or yet better - your second solution (create `elements|atoms|components|helpers` file and put there every styled component that is unique to this FC and will not be useful globally.